digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/ClientUtils.scala

```diff
@@ -6,16 +6,20 @@ import scala.annotation.tailrec
 object ClientUtils {
 
   @tailrec
-  def retry[T](retries: Int, logger: Logger)(fn: => T): T = {
+  def retry[T](retries: Int, logger: Logger,exponentToLoop: Int =1)(fn: => T): T = {
+    var exponent = exponentToLoop
```

Comment:
Mitä hyötyä tällä muutoksella haetaan?

Reply:
joo pitäis kyl kirjoittaa pull request kuvaus eka ennen kuin avaa tai kutsuu ihmisiä, vähän laiskana en jaksa aina kirjoitella mitään kuvauksia. Pitää vähän harjoitella alkaa kirjoitella niitä.

Reply:
Vähän mietin, että auttaako retryjen lkm lisääminen. Muistaakseni lokeissa näkyi, että kaikki Viite kyselyt jäivät X ajaksi jumiin kunnes niiden vastaukset palautuivat n. 14min jälkeen ja yhteys palasi toimimaan taas normaalisti.

Reply:
Tämä taitaa hidastaa taas integraatiorajapintaa ainakin irrotusten aikana, jos viite yhteydessä on ongelmia ja ruvetaan jokaisen kyselyn kohdalla odottamaan enemmän aikaa sekä uudelleen yrityksiä olisi nyt viisinkertainen määrä. Ei tuo odotusaika ole mikään mahdottoman pitkä, mutta pitäisikö kuitenkin jättää retryjen määrä pienemmäksi esim. 3 tai 5 kpl?

Reply:
joo tällä hetkellä ne jotkut kaatuu siihen toiseen kyselyyn joten varmaan viis ok.